package.json

```diff
@@ -34,6 +35,7 @@
     "@babel/preset-react": "^7.0.0",
     "babel-jest": "^24.8.0",
     "babel-loader": "^8.0.6",
+    "csv-write-stream": "^2.0.0",
```

Comment:
It's always great to add your common bash commands as scripts in package.json. Nice.